scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -219,6 +226,29 @@ workflow CNVSomaticPairWorkflow {
             gatk_docker = gatk_docker,
             disk_space_gb = plot_normal_disk
     }
+
```

Comment:
Not in this commit, but can you add white space after the PlotDenoisedCopyRatiosTumor task above?

Reply:
I think I did it.  I was not 100% sure that I knew what you were indicating.